knotx-standalone/src/main/resources/logback.xml

```diff
@@ -26,8 +26,22 @@
     </encoder>
   </appender>
 
+  <appender name="access" class="ch.qos.logback.core.FileAppender">
+    <file>access.log</file>
+    <append>true</append>
+    <!-- set immediateFlush to false for much higher logging throughput -->
+    <immediateFlush>true</immediateFlush>
+    <encoder>
+      <pattern>%msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <logger name="io.vertx.ext.web.handler.impl.LoggerHandlerImpl" level="INFO" additivity="false">
+    <appender-ref ref="access"/>
+  </logger>
+
   <logger name="io.knotx" level="INFO"
```

Comment:
Can we add logger for io.vertx, io.reactivex ?

Reply:
Let's do that for `WARN` level.